h2o-core/src/test/java/water/rapids/ast/prims/mungers/AstRelevelByFreqTest.java

```diff
@@ -10,7 +10,7 @@ public class AstRelevelByFreqTest {
 
     @Test
     public void takeTopN() {
-        assertArrayEquals(new int[]{6, 5, 4, 0, 3, 2, 1}, AstRelevelByFreq.takeTopN(new int[]{5, 4, 3, 2, 1}, 3, 7));
-        assertArrayEquals(new int[]{6, 0, 5, 4, 3, 2, 1}, AstRelevelByFreq.takeTopN(new int[]{5, 4, 3, 2, 1}, 5, 7));
+        assertArrayEquals(new int[]{5, 4, 3, 0, 1, 2, 6}, AstRelevelByFreq.takeTopN(new int[]{5, 4, 3, 2, 1}, 3, 7));
```

Comment:
Can we add here messages in case it fail and add test with domainSize < arraysize, domainSize == arraysize to easily see the difference. 

And test if it is actually sorting, I see in both cases that input is already sorted.

Reply:
sorry, the topN means number of untouched. It would nice to catch it in the test.

Reply:
I removed the domainSize parameter. It was no longer helpful.

The input is not sorted ASC, so we test the sorting works.  Current example input is: {5, 4, 3, 2, 1, 0, 6}.